### DISCUSSION TITLE
custom url path

### DISCUSSION THREAD

**Discussion by atutr001** [Hubro/clipface#48] [Q&A] [answered]
Hi 
right now the docker container doesn't have an option for choosing a URL path (default / | https://clipface.com/) If possible I would like to use a different URL path like https://clipface.com/customname 
Thanks.

**Comment by Hubro** [comment, marked as answer]
This isn't currently possible, unfortunately, but an issue is already open here: https://github.com/Hubro/clipface/issues/44
